etc/org.opencastproject.organization-mh_default_org.cfg

```diff
@@ -32,16 +32,22 @@ id=mh_default_org
 #
 name=Opencast Project
 
-# Server name that identifies the organization (tenant) on this Opencast installation.
+# Server names that identifies the organization (tenant)
 #
-# Use the public url that is assigned to this tenant. In out-of-the-box installations
-# with only one tenant, this value corresponds to the server.url in etc/config.properties.
-# Make sure to adjust these two keys accordingly:
+# Use the public domain names which are assigned to this tenant. Multiple values can be separated by comma. The final
+# set of domain names identifying this organization within Opencast will be the union of all domains defined on all
+# nodes.
 #
-#   - prop.org.opencastproject.admin.ui.url
-#   - prop.org.opencastproject.engage.ui.url
+# The usage of this is to map a request to a specific organization. For example:
```

Comment:
I'm not sure this example is entirely clear to me. Maybe expand it a bit? Trying it myself, though, makes this comment super long which is also not ideal so maybe it's fine. :thinking: